legacy/src/app/directives/script_select.js

```diff
@@ -17,6 +17,12 @@ function filterScriptsByParam(scripts, param) {
   });
 }
 
+const sortScripts = (a, b) => {
```

Comment:
This is fine (given we're killing this soon), but just so you know there's a nicer way to do this today with [localCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#Browser_compatibility):

```
['qux', 'bar', 'foo'].sort((a, b) => a.localeCompare(b))
> ["bar", "foo", "qux"]
```

Reply:
I'll remember for next time!